Gallery.Api/Controllers/ExhibitController.cs

```diff
@@ -172,6 +199,10 @@ public async Task<IActionResult> Create([FromBody] Exhibit exhibit, Cancellation
         [SwaggerOperation(OperationId = "copyExhibit")]
         public async Task<IActionResult> Copy(Guid id, CancellationToken ct)
         {
+            if (!await _authorizationService.AuthorizeAsync([SystemPermission.CreateExhibits], ct) ||

```

Comment:
can combine these?

Reply:
I don't think these can be combined?  The user has to be able to create exhibits AND has to be able to view the requested exhibit.  The Authorize method first checks for all system permissions and then for all exhibit permissions.  In this case the user could have a create SystemPermission and a view ExhibitPermissions.  Combining the checks wouldn't allow that situation.  Am I understanding this correctly?

Reply:
You're right in this case, sorry!